presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -571,24 +574,21 @@ public void testArrayTimestamp(boolean legacyTimestamp, boolean insertWithPresto
             dataType = arrayDataType(timestampDataType(), "timestamp[]");
             dataSetup = postgresCreateAndInsert("tpch.test_array_timestamp");
         }
-        // using two non-JVM zones so that we don't need to worry what Postgres system zone is
         for (ZoneId sessionZone : ImmutableList.of(ZoneOffset.UTC, jvmZone, vilnius, kathmandu, ZoneId.of(TestingSession.DEFAULT_TIME_ZONE_KEY.getId()))) {
             DataTypeTest tests = DataTypeTest.create()
                     .addRoundTrip(dataType, asList(beforeEpoch))
                     .addRoundTrip(dataType, asList(afterEpoch))
                     .addRoundTrip(dataType, asList(timeDoubledInJvmZone))
                     .addRoundTrip(dataType, asList(timeDoubledInVilnius));
 
-            if (!insertWithPresto) {
-                // when writing, Postgres JDBC driver converts LocalDateTime to string representing date-time in JVM zone
-                // TODO upgrade driver or find a different way to write timestamp values
-                addTimestampTestIfSupported(tests, legacyTimestamp, sessionZone, epoch); // epoch also is a gap in JVM zone
-                addTimestampTestIfSupported(tests, legacyTimestamp, sessionZone, timeGapInJvmZone1);
-                addTimestampTestIfSupported(tests, legacyTimestamp, sessionZone, timeGapInJvmZone2);
+            if (!ZoneId.systemDefault().getId().equals("America/Bahia_Banderas")) {
```

Comment:
This should always evaluate to false, because we use a fixed system time zone in tests.

Why did you replace the `if (!insertWithPresto)` condition?

Reply:
> This should always evaluate to false, because we use a fixed system time zone in tests.

Correct. It's there just to emphasise that shifting by 1-hour is something unique for the combination of current JVM timezone and specific timestamps. 
Maybe it's worth adding a comment on that. 

> Why did you replace the `if (!insertWithPresto)` condition?

The shifting by 1-hour is happening also when inserting with Presto, so in order not to completely remove those tests, I added the condition on the JVM timezone.

Reply:
This is dead code, even thought it doesn't look as such. 
Please add it as a regular `.addRoundTrip(...` cases, but keep it commented out with proper explanation